src/pages/viewer-app/e2e.viewer-app.spec.ts

```diff
@@ -0,0 +1,41 @@
+import puppeteer from 'puppeteer';
+
+describe('ViewerApp', () => {
+  let browser;
+  let page;
+
+  beforeAll(async () => {
+    browser = await puppeteer.launch();
+    page = await browser.newPage();
+    await page.goto('http://localhost:3000/#/viewer');
+  });
+
+  afterAll(() => browser.close());
+
+  it('Contains control panel', async () => {
+    await page.waitForSelector('#control-panel');
+    await page.select('#base-map', 'Light');
+    const controlOptions = [
+      'New York',
+      'San Francisco v1.6',
+      'San Francisco v1.7',
+    ];
+
+    for await (const item of controlOptions) {
+      expect(await page.select('#tileset', item)).toEqual([item]);
```

Comment:
Will be good to detect that tileset really has been changed not only in selector.
WE can't detect it in canvas but, memory stats widget displays tcurrent ileset url.
You can check it.